docs/reference/supervisor.md

```diff
@@ -106,6 +112,7 @@ node = wb_supervisor_node_get_selected()
 | `/supervisor/get_self` | `service` | [`webots_ros::get_uint64`](ros-api.md#common-services) | |
 | `/supervisor/get_from_def` | `service` | `webots_ros::supervisor_get_from_def` | `string name`<br/>`uint64 proto`<br/>`---`<br/>`uint64 node` |
 | `/supervisor/get_from_id` | `service` | `webots_ros::supervisor_get_from_id` | `int32 id`<br/>`---`<br/>`uint64 node` |
+| `/supervisor/get_from_device` | `service` | `webots_ros::supervisor_get_from_string` | `string value`<br/>`---`<br/>`uint64 node` |
```

Comment:
The argument name is `value` not `string`.
We use this kind of argument name in many other 
Personally I think we should avoid create too specific services otherwise we cannot reuse them and we have to create new ones for each function.

Reply:
Actually I think that a better name would be `node_get_from_string`.
But this won't be coherent with the other supervisor messages.

Reply:
OK, so let's keep it like that...